### ISSUE TITLE
Network-based distribution?

### ISSUE THREAD

**Issue by lofidevops** [copyleft-next/copyleft-next#51]
I currently use the AGPLv3.0-or-later. Ideally, copyleft-next itself would have an AGPL-like clause. Given that this is still a source of concern for *some people*, how about an (optional?) clause that would trigger AGPL-like distribution rules?

In SPDX terms, I'm guessing it would be a distinct license identifier: `copyleft-next` and `copyleft-next-network`

**Comment by Seirdy** [comment]
Agreed; the network-usage loophole that enables SaaS basically lets vendors give users "thin clients" to run software remotely, out of their control; this defeats the point of free software.
